eq-author/src/components/RichTextEditor/index.js

```diff
@@ -113,6 +115,18 @@ const getContentsOfPipingType = type => contents =>
 const getAnswerPipes = getContentsOfPipingType("answers");
 const getMetadataPipes = getContentsOfPipingType("metadata");
 
+const filterEmptyTags = value => {
+  if (!isNull(cheerio(trim(value)).html())) {
```

Comment:
What is this first check doing?

Reply:
It makes sure there is html in the value otherwise the cheerio.text on the next line wont work

Reply:
Can we give it a name then like - `isHtml`?